src/site/apt/user_query_hdfs.apt

```diff
@@ -0,0 +1,120 @@
+~~ Licensed to the Apache Software Foundation (ASF) under one or more
+~~ contributor license agreements.  See the NOTICE file distributed with
+~~ this work for additional information regarding copyright ownership.
+~~ The ASF licenses this file to You under the Apache License, Version 2.0
+~~ (the "License"); you may not use this file except in compliance with
+~~ the License.  You may obtain a copy of the License at
+~~
+~~     http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing, software
+~~ distributed under the License is distributed on an "AS IS" BASIS,
+~~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~~ See the License for the specific language governing permissions and
+~~ limitations under the License.
+
+Executing a Query in HDFS
+
+
+* 1. Connecting VXQuery with HDFS
+
+    The only variable VXQuery needs in order to access HDFS is the path to the HDFS configuration directory.
+
+    We define that path in the <<cluster.properties>> file that is located in <<vxquery-root/vxquery-server/src/resources/conf/>> directory.
+
+    Inside that file we need to change the value of the variable <<HDFS_CONF>> to the full system path of the HDFS configuration.
+
+    For example, <HDFS_CONF=/usr/local/hadoop/etc/hadoop> defines that the configuration files for our HDFS installation are located in </usr/local/hadoop/etc/hadoop> path.
+
+
+* 2. Running the Query
+
+  For files stored in HDFS there are 2 ways to access them from VXQuery.
+
+  [[a]] Reading them as whole files.
+
+  [[b]] Reading them block by block.
+
+** a. Reading them as whole files.
+
+  For this option you do not need to change your query at all. VXQuery will read the path of the files you request in your query and try to locate them 
+
+  1) First in your local file system
+
+  and if the path does not exist there,
```

Comment:
The way they recommend needs hdfs to exists in order to check which will not work for systems without hdfs,I dont think it suits us.For our scenario it seems better to first check the local filesystem.